### PR TITLE
Makes some requested changes to mood

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd
del: removed beauty / dirtyness
balance: Mood no longer gives you hallucinations, instead makes you go into crit sooner
/:cl:

oranges requested these changes, i havn't coded in DM for a while so I might have made some mistakes

### CODE REVIEW

File: code/datums/brain_damage/special.dm [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	if(prob(4))
 		if(prob(33) && (owner.IsStun() || owner.IsKnockdown() || owner.IsUnconscious()))
 			speak("unstun", TRUE)
-		else if(prob(60) && owner.health <= HEALTH_THRESHOLD_CRIT)
```
Comment (deathride58) [inline code/datums/brain_damage/special.dm]: Why not just use a is_in_crit() proc or a define or something similar? You copy and paste the `+ owner.crit_modifier()` quite a lot

Comment (Qustinnus) [reply code/datums/brain_damage/special.dm]: proc is overhead

Comment (optimumtact) [reply code/datums/brain_damage/special.dm]: \>calls procs overhead
\>adds his own proc to calculate modifiers
scumbag steve

Comment (Qustinnus) [reply code/datums/brain_damage/special.dm]: image
you ignored me after that :(

Comment (optimumtact) [reply code/datums/brain_damage/special.dm]: how come it dosn't show my owner status bad

Comment (ExcessiveUseOfCobblestone) [reply code/datums/brain_damage/special.dm]: u r a sick sick girl.

**Comment by Wilchenx** [comment]
>instead makes you go into crit sooner

as in less soft-crit?

**Comment by optimumtact** [comment]
you enter soft crit earlier, that threshold is usually zero but it will get bumped up slightly

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
The `crit_modifier` should be on the mob (physiology would be a nice holder) and your mood should be affecting that when it hits your desired threshold under `update_mood`.

**Comment by epicmemename** [comment]
If you wanna fit the title just revert moods as a whole

File: code/modules/mob/living/living_defense.dm [hunk 2]
```diff
@@ -61,6 +61,9 @@
 		else
 				return 0
 
+/mob/living/proc/crit_modifier()
+	return 0
```
[the remark this answers is not in the document]
Comment (Cyberboss) [reply code/modules/mob/living/living_defense.dm]: `HEALTH_THRESHOLD_CRIT`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
come on @Qustinnus

**Comment by Qustinnus** [comment]
working on it

**Comment by Wilchenx** [comment]
>removed beauty / dirtyness

I'd rather have it as a trait if you can bother redoing it into one.

**Comment by Qustinnus** [comment]
Depends on if the fruit is okay with that

**Comment by PKPenguin321** [comment]
Maybe make the mood alert face not show up until it dips into negatives to appease the OCD whiners

**Comment by optimumtact** [comment]
lol how about you finish the PR

**Comment by Wilchenx** [comment]
>daddy oranges

floyd why

**Comment by Qustinnus** [comment]
idk i just woke up, my commit messages are pure brainfarts

**Comment by ExcessiveUseOfCobblestone** [comment]
> he merged those crit_modifier procmemes

sick sick fruit

**Pull request merged**